### PR TITLE
mrc-2349 remove references to specific custom fields

### PR DESCRIPTION
**Pull request by hillalex**
* new migration that recreates the `orderlyweb_report_version_full` view
* regenerated db classes from an instance of orderly without custom fields

There is no good way to target this issue via tests, since we don't have a way of bringing up `orderly` instances with different configs. To test locally is a bit fiddly, involves bringing up the project, then manually modifying the `demo` orderly folder (deleting all existing reports, modifying the yml and re-running a report or two)

Must be merged/deployed in tandem with https://github.com/vimc/orderly/pull/167

### CODE REVIEW

**Comment by hillalex** [comment]
This is now failing for the same reason as `master`, waiting on that to be fixed before this can be reviewed.

File: buildkite/make-db.sh
Comment (richfitz) [inline buildkite/make-db.sh:3]: once https://github.com/vimc/orderly/pull/167 is merged, revert this

File: config/orderly_server_version
Comment (richfitz) [inline config/orderly_server_version:1]: and revert this

File: src/app/build.gradle
Comment (richfitz) [inline src/app/build.gradle:76]: more branch pins (can this not come from the config/orderly_server_version, in some future maintenance PR?)

**Review by richfitz — APPROVED** [review]
LGTM with the pins updated after the orderly merge

File: src/app/build.gradle
Comment (hillalex) [reply src/app/build.gradle:76]: yeah this whole test setup is a mess. why is this a gradle task, that calls shell commands, that it itself utlimately called by a shell script?

**Pull request merged**
